### ISSUE TITLE
Empty box of chematics and same for material list

### ISSUE THREAD

**Issue by Arkusis** [maruohon/litematica#912]
Wanted to test if all is working with few schematics to use later but no sadly as can see down.
Tried few thinks to work it out but still same think. Its not only this one cause I tried few more schems to see if they will work but it happens for all of them so no idea what else to do with it
![2024-11-28_10 50 58](https://github.com/user-attachments/assets/294767b3-360c-4fd8-afa3-f9120d792528)
![2024-11-28_10 50 43](https://github.com/user-attachments/assets/a2e2c314-cf17-4470-96ec-e13aa11ca4b7)

**Comment by maruohon** [comment]
It seems like you have a non-`.litematic` schematic (since it says `Converted schematic`), so it's possible the format wasn't recognized correctly or it's broken somehow. Can you send some of those schematic files? Where are they from?

**Comment by Arkusis** [comment]
Yes these were converted schematics from minecraft-schematic which I have done in previous version but cant get to work now.
They are from minecraft-schematics
Maybe Im dumb or they just dont work with it
Did these as tests
https://www.minecraft-schematics.com/schematic/8481/
https://www.minecraft-schematics.com/schematic/15974/
https://www.minecraft-schematics.com/schematic/8430/
https://www.minecraft-schematics.com/schematic/13466/

**Comment by maruohon** [comment]
I think this was the issue of broken `.schematic` format import in some of the earlier 1.21.x mod versions. It should be fixed in later releases.
